scripts/load_csv_prereg.py

```diff
@@ -0,0 +1,100 @@
+""" Load pre-reg questions into schema from csv file
+"""
+
+import csv
+import urllib2
+import logging
+import os
+import json
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+""" Given a question number, returns list containing page id, page index, and index of question on page
+"""
+def getPage(quest_num):
+	pageDir = {'page1':['01', '02', '03', '04'], 'page2':['05', '06', '07', '08'], 'page3':['09', '10', '11', '12', '13', '14', '15'], 'page4':['16', '17', '18', '19', '20', '21'], 'page5':['22']}
```

Comment:
I'm okay with the hard coded file name, but am not clear why the pages need to be hard coded? 

Also the indentation on this file is very odd. Please make sure your editor is configured to use 4 spaces for indents.